src/main/java/com/byznass/tiolktrack/kernel/dao/LocationPersister.java

```diff
@@ -1,8 +1,9 @@
 package com.byznass.tiolktrack.kernel.dao;
 
+import com.byznass.tiolktrack.kernel.TiolkTrackException;
 import com.byznass.tiolktrack.kernel.model.Location;
 
 public interface LocationPersister {
 
-	void persistLocation(Location location);
+	void persistLocation(Location location) throws TiolkTrackException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.byznass.tiolktrack.kernel.TiolkTrackException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)